docs/how_to/aggregator.md

```diff
@@ -0,0 +1,159 @@
+# Setting up m3aggregator
+
+## Introduction
+
+m3aggregator is used to cluster stateful downsampling and rollup of metrics before they are store in M3DB. The m3coordinator also performs this role but is not cluster aware. This means metrics will not get aggregated properly if you send metrics in round round fashion to multiple m3coordinators for the same metrics ingestion source (e.g. Prometheus server).
```

Comment:
store -> stored

Reply:
We usually say m3coordinator not "the m3coordinator"

Reply:
"m3coordinator can also perform this role, but is not clustered. This makes it difficult to scale out as metrics will not get aggregated properly when using round robin load balancing across multiple m3coordinators (since aggregation is a stateful operation)" or something like that

Reply:
"round round fashion" -> "round robin fashion"